docs/cli/index.md

```diff
@@ -17,6 +17,8 @@ overview of Gemini CLI, see the [main documentation page](../index.md).
 - **[Keyboard Shortcuts](./keyboard-shortcuts.md):** A reference for all
   keyboard shortcuts to improve your workflow.
 - **[Tutorials](./tutorials.md):** Step-by-step guides for common tasks.
+- **[Alternate Screen Buffer](./alternate-buffer.md):** Enable a flicker-free,
```

Comment:
Let's actually put this under "Experimental Features" and place that after "Advanced Features"